Snowflake.Data.Tests/SFConnectionIT.cs

```diff
@@ -312,5 +312,21 @@ public void TestOktaConnection()
                 }
             }
         }
+        [Test]
+        [Ignore("This test requires manual interaction and therefore cannot be run in CI")]
+        public void TestSSOConnection()
+        {
+            using (IDbConnection conn = new SnowflakeDbConnection())
+            {
+                conn.ConnectionString = "scheme=http;host=testaccount.reg.snowflakecomputing.com;port=8082;user=qa@snowflakecomputing.com;password=Test123!;" +
+                    "account=testaccount;role=sysadmin;db=testdb;schema=public;warehouse=regress;authenticator=externalbrowser";

```

Comment:
Move these parameter out to `parameters.json`
See JDBC' ConnectionManual.java for the environment variable names.

Reply:
```
export SNOWFLAKE_TEST_ACCOUNT=ACCOUNT
export SNOWFLAKE_TEST_SSO_USER=USER
export SNOWFLAKE_TEST_PORT=PORT
export SNOWFLAKE_TEST_SSL=SSL
export SNOWFLAKE_TEST_DATABASE=DATABASE
export SNOWFLAKE_TEST_SCHEMA=SCHEMA
```

Also remove password from the string.